sdk/sdk/Models/dao/OBAModelDAOUserPreferencesImpl.m

```diff
@@ -243,8 +235,6 @@ - (void) writeSetRegionAutomatically:(BOOL)setRegionAutomatically {
     NSUserDefaults * user = [NSUserDefaults standardUserDefaults];
     NSNumber * v = @(setRegionAutomatically);
     [user setObject:v forKey:kSetRegionAutomaticallyKey];
-
-    [[GAI sharedInstance].defaultTracker set:[GAIFields customDimensionForIndex:2] value:(setRegionAutomatically ? @"YES" : @"NO")];
```

Comment:
I'll create a separate issue for that sometime soon for discussion.  Essentially it is about abstracting the details of the implementation (i.e. how it persists stuff, where it logs analytics) and separating concerns (i.e. a class is only responsible for persist or analytics, not both).  In its simplest form, there would be another class created behavior of which would be composed of GAI and persistance classes behaviours.  

It leads to a more broad topic of how you would want to architect the application, and for which items to optimize.